From e4ac05da6d4ffa8caecc14c1235d51ee0609e409 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 12 Jan 2012 13:00:24 +0800
Subject: ENGR00171462-1 vpu: add BIT_PIC_RUN register in mxc_vpu.h

Add BIT_PIC_RUN register definition in mxc_vpu.h of arch/arm

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc_vpu.h |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_vpu.h b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
index dd162a2..da4f39d 100755
--- a/arch/arm/plat-mxc/include/mach/mxc_vpu.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2011 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2004-2012 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -60,8 +60,9 @@ struct vpu_mem_desc {
 
 #define BIT_CODE_RUN			0x000
 #define BIT_CODE_DOWN			0x004
-#define	BIT_INT_CLEAR			0x00C
-#define	BIT_INT_STATUS			0x010
+#define BIT_INT_CLEAR			0x00C
+#define BIT_INT_STATUS			0x010
+#define BIT_CUR_PC			0x018
 #define BIT_INT_REASON			0x174
 
 #define MJPEG_PIC_STATUS_REG		0x3004
-- 
1.7.9.5

